[Security][9.5 & Serverless] Document the new flyout system for alert and event details (#7719)
elastic/docs-content: solutions/security/advanced-entity-analytics/view-entity-details.md

Reviewer: Should we replace "expand" here?
```suggestion
Open the section's child flyout to access additional details about the entity's risk inputs.
```

## Entity details flyout

> Refer to [Details flyouts](/solutions/security/get-started/elastic-security-ui.md#details-flyouts) for the flyout's shared header controls, footer, and child flyout navigation.

Reviewer: ```suggestion
Refer to [Details flyouts](/solutions/security/get-started/elastic-security-ui.md#details-flyouts) for the toolbar controls, footer, and child flyout navigation shared across {{elastic-sec}} flyouts.
```
Matches the wording from [manage-discoveries-from-attacks-page.md](https://github.com/elastic/docs-content/pull/7719/changes#diff-fb2f1d26a635e103869033764214e6afa7e3268918b6e38686cc18690024599d) and [view-detection-alert-details.md‎](https://github.com/elastic/docs-content/pull/7719/changes#diff-d54eb6ea3932d8d2efd92abe7e5709eeaa9d04194ee3b8b4281e92425d7baafe), which I prefer slightly since it states what surface the flyout is shared across.

Author: Fixed in [`3ecdec3` (this PR)](https://github.com/elastic/docs-content/pull/7719/changes/3ecdec3bbc26623828cfa973fd1891ad6db7ed23).

serverless: ga
```

The **Resolution** section shows whether the entity belongs to a [resolution group](/solutions/security/advanced-entity-analytics/entity-resolution.md). Click **Resolution group** to open a child flyout that displays all entity records linked to this entity — including the primary entity and any aliases — with their entity name, ID, source, and risk score.

To add an entity to the group, search by entity name or ID in the **Add entities to resolution group** table and click the add icon {icon}`plus_in_circle` next to the entity you want to link. To remove an entity from the group, click **X** {icon}`cross` in the **Actions** column of the **Resolution group** table. Entities must be removed individually.

### Insights
```yaml {applies_to}
stack: ga 9.5+
serverless: ga

> ```

Reviewer: ```suggestion
### Insights
```
This section isn't new in 9.5, it existed pre-9.0
